src/elements/core/styles/mixins/table.scss

```
@@ -0,0 +1,135 @@
+@use './typo' as t;
+
+/*
+ * SBB table mixin
+ * Notes:
+ * We cannot use `border-collapse` because it is not compatible with the `border-radius` property
+ */
+@mixin table {
+  @include table-striped;
+  @include table-m;
+
+  --sbb-table-border: 1px solid var(--sbb-table-border-color);
```

Couldn't we use the var?
```suggestion
  --sbb-table-border: var(--sbb-border-width-1x) solid var(--sbb-table-border-color);
```